services/app/src/hooks/useChessBoard.js

```diff
@@ -54,6 +54,12 @@ export const useChessBoard = () => {
     setBoard(copyOfBoard);
   }
 
+  const insertMoveScoreIntoBoard = (turn) => {
```

Comment:
this can be taken from the chessboard context

Reply:
I think `turn` prop makes more sense, because we need to access the cell of the chessboard, in which we insert the score.